platforms/common/include/px4_platform_common/px4_work_queue/WorkQueueManager.hpp

```diff
@@ -49,6 +49,7 @@ struct wq_config_t {
 namespace wq_configurations
 {
 static constexpr wq_config_t rate_ctrl{"wq:rate_ctrl", 1664, 0}; // PX4 inner loop highest priority
+static constexpr wq_config_t ctrl_alloc{"wq:ctrl_alloc", 9500, 0}; // PX4 control allocation, same priority as rate_ctrl
```

Comment:
Have you verified this recently?

Reply:
Here it is with full 6x16 matrix inversion on fmu-v4.
![control_alloc_top](https://user-images.githubusercontent.com/1379463/84738637-91953f80-afaa-11ea-9e35-60c8d4f37a64.png)
There is a fair bit of optimization to do.